### ISSUE TITLE
[Feature]: Collection Lock Setting

### ISSUE THREAD

**Issue by DreamStatic** [Kometa-Team/Kometa#1131]
### Is your feature request related to a problem? Please elaborate.

I recently noticed a new Plex collection that was NOT auto created when I added a new movie that was part of a 2 movie collection. Normally, Plex would create the collection when adding the movie and then it would show in the Movie details. Which lets me know it is part of a series of movies. 

After further investigation, I realized the issue. I utilize PMM to build collections based on actors, which one of them starred in this new movie that is a sequel. When looking at the movies in this actor's collection, I noticed the "Collections" field under Edit was locked. So Plex could not add the Movie series collection to the new movie.

As a test, I removed the lock to the Edit - Collections field. Then ran a metadata refresh on the new movie. Even though previously I had ran multiple metadata refreshes, before realizing the lock was present, now with the lock removed it did append the movie series collection in addition to the existing actor name collection. 

### Describe the solution you'd like

I would like to not "LOCK" the collection field, and allow Plex to append additional collections to a single movie. Currently, I see PMM does add multiple collection IDs to a given movie, if I have them in multiple PMM built collections. i.e. Multiple Actor collections, Box Office, Animated, etc.  Additionally, it appears it appends to any existing collections when building new PMM collections to a given movie. i.e. appends my custom PMM collections to existing Plex movie series collections. 

In short, I want my cake and eat it too. :) I would enjoy having custom PMM collections that are updated when PMM is triggered to update, and have Plex add auto movie series collections when adding new movies. As I would prefer not to have to manually manage movie series collections since Plex handles that pretty well automatically.

**Comment by meisnate12** [comment]
so plex will not append the collection it will once update the whole field. If you dont lock the collection field then if the metadata refreshes the collection will be removed from other collections say an actor collection

**Comment by fjeddy** [comment]
> so plex will not append the collection it will once update the whole field. If you dont lock the collection field then if the metadata refreshes the collection will be removed from other collections say an actor collection

So then the question becomes; can Plex Meta Manager replicate the Plex behavior and add movies that belongs in the same series to a collection?

I didn't realize this was the case when I setup Plex Meta Manager, you are technically breaking a Plex feature, and there should be a HUGE warning.

**Comment by honkerst** [comment]
I'd like to request this be revisited please. 

Plex no longer erases the Kometa tags when you refresh metadata, so could we please have some way of leaving the collections field unlocked after a Kometa run.

**Comment by antwanchild** [comment]
> I'd like to request this be revisited please.
> 
> Plex no longer erases the Kometa tags when you refresh metadata, so could we please have some way of leaving the collections field unlocked after a Kometa run.

Collections don't have the option to refresh metadata, and collections are the only thing that has the Kometa label.

**Comment by honkerst** [comment]
I dont think that's true, or maybe we're talking about diffferent things. Try it yourself, find a film that has Kometa collections, but doesn't have the Plex-added 'sequels' collection, unlock the collectiuons field and refresh metadata for that film. It'll add the Plex 'sequels' collection (assuming you have the sequel in your library), but leave your Kometa collections there.

**Comment by antwanchild** [comment]
The “Kometa” label you are talking about because I can only go off what you say, you must be talking about some case related the plex auto collections. I turn those off and let Kometa handle my franchise collections. Or any collections for that matter. 
but the “Kometa” label exists only on collections, not on individual movies. 
if you are talking about other types of labels, then those may exist from Kometa, but I think the behavior is for plex to lock them after they are edited

**Comment by honkerst** [comment]
Yeah, I didn't specifically mean the label named 'kometa', just the tags etc that Kometa creates. 

For anyone else who arrives here via Google like I did, I have tmeporarily fixed my problem by mass-selecting 7300 movies (this is quite tedious), and batch editing them to lock the collections field, and then mass-selcting them all again, and batch unlocking the collections field. Then refresh metadata on the whole library, and hey presto 12 hours later I have Kometa collections, and Plex auto-collections. 

In future I will have just have to remember to manually unlock collections and refresh metadate for each newly added movie.

**Comment by antwanchild** [comment]
To be honest, most people who use Kometa, turn plex auto collections off because it allows more control of the collections

**Comment by honkerst** [comment]
I have around 150 collections defined in Kometa, it's my favourite part of the software :) 

It's just that when I click into a movie details screen I want to see the sequels listed in a collection but I don't want to create 500+ movie series collections if Plex will do it automatically and add a nice collection poster so I don't have to make/find one.

**Comment by antwanchild** [comment]
Kometa has a default called “franchise” and it creates these collections. You don’t have to create any of them. And doing in Kometa allows you to use some template variables to change and / or modify parts of your like

**Comment by honkerst** [comment]
Oh that's interesting! Not sure if that existed when I set up my config, I shall read up on it, cheers.

**Comment by antwanchild** [comment]
It’s been part of Kometa for several years

Sent from my iPhone, please forgive any typos or misspellings.
Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
